src/runtime/pythonengine.cs

```diff
@@ -469,61 +469,15 @@ public static void EndAllowThreads(IntPtr ts)
         }
 
 
-        /// <summary>
-        /// ImportModule Method
-        /// </summary>
-        /// <remarks>
-        /// Given a fully-qualified module or package name, import the
-        /// module and return the resulting module object as a PyObject
-        /// or null if an exception is raised.
-        /// </remarks>
-        public static PyObject ImportModule(string name)
```

Comment:
We should probably leave these for backward-compat with an `[Obsolete]` annotation. Since they can all be easily implemented in terms of the "new" way of doing things, this is very little additional code to maintain and we can drop it with 3.1 or so.